cmd/main.go

```diff
@@ -18,7 +18,7 @@ const (
 )
 
 var (
-	configFileFlag = cli.StringFlag{
+	configFileFlag = cli.StringSliceFlag{
 		Name:     config.FlagCfg,
 		Aliases:  []string{"c"},
 		Usage:    "Configuration `FILE`",
```

Comment:
Should it be a plural now?
```suggestion
		Usage:    "Configuration `FILES`",
```